presto-raptor/src/test/java/com/facebook/presto/raptor/systemtables/TestShardMetadataRecordCursor.java

```diff
@@ -93,20 +99,36 @@ public void teardown()
     }
 
     @Test
-    public void testSimple()
+    public void testDateTemporalColumnNoFilter()
```

Comment:
Good point... I should keep it because it covers different predicates on the temporal column.